fairlearn/reductions/_moments/weighted_error_rate.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import numpy as np
+from itertools import repeat
+from .moment import LossMoment,ClassificationMoment,_ALL, _LABEL
+from .bounded_group_loss import SquareLoss
+_WEIGHTS = "weights"
+
+class WeightedErrorRate(LossMoment):
+    """Weighted Loss error."""
+    ## Not Sure about whether this should inherit from ClassificationMoment or LossMoment
+
+    short_name = "Weighted Error Rate"
+    def __init__(self,loss):
+        super(WeightedErrorRate,self).__init__(loss)
+
+    ## for what we need here is augmented data. Hence to avoid unnecessary calculation, we use augmented data having been calculated in regression_moment here
```

Comment:
The first sentence isn't a real sentence. Please rewrite that.
Also, please stay within 99 characters for every line, and for comments perhaps even just 79. I believe flake8 will complain about the former of those rules.

nit: typo inferface